docs/blockio.md

```diff
@@ -12,4 +12,4 @@ CRI Resource Manager applies block IO contoller parameters to pods via
 
 ## Configuration
 
-See [sample blockio configuration](../sample-configs/blockio.cfg).
```

Comment:
It would have been nicer to distill this (and other similar fixes to the raw docs) into a separate 'docs: documentation fixes.'-commit. But it's fine like this now, no need to split it out any more.